src/common/single_threaded_blas.cpp

```diff
@@ -15,10 +15,14 @@
 #include <mkl_service.h>
 #endif
 
+#include <dlaf/common/assert.h>
```

Comment:
Does this need `#ifdef DLAF_ASSERT_MODERATE_ENABLE`?

Reply:
Good point. Not necessary, but it doesn't hurt. I added the `ifdef`.